Models/DTOs/ArmourDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Monster_Builder_Web_API.Models.DTOs;
+
+//I used a DTO because there was a fair bit of logic in the Armour class
+public class ArmourDTO
```

Comment:
If the update DTO ends up looking different which it might well do then this needs a rename